databroker/assets/core.py

```diff
@@ -168,6 +168,7 @@ class _PrivateException(Exception):
 def insert_resource(col, spec, resource_path, resource_kwargs,
                     known_spec, root, path_semantics='posix', uid=None,
                     run_start=None,
+                    id=None,
```

Comment:
Maybe worth to combine it with the previous line?

Reply:
Do you mind to reformat the signature, so it's more compact, as suggested above or even better?

Reply:
Addressed.